src/gui/executionlog.cpp

```diff
@@ -35,18 +35,22 @@
 #include <QPalette>
 #include "executionlog.h"
 #include "ui_executionlog.h"
-#include "base/logger.h"
+#include "base/preferences.h"
 #include "guiiconprovider.h"
 #include "loglistwidget.h"
 
 ExecutionLog::ExecutionLog(QWidget *parent)
     : QWidget(parent)
     , ui(new Ui::ExecutionLog)
-    , m_msgList(new LogListWidget(MAX_LOG_MESSAGES))
-    , m_peerList(new LogListWidget(MAX_LOG_MESSAGES))
+    , m_peerList(new LogListWidget(MAX_LOG_MESSAGES, true, true, true, true))
```

Comment:
By and large, this class should not contain any logic associated with handling log messages (and only the initialization/configuration of interface elements). This should be moved to the appropriate widget code.